oxidation/bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -84,17 +84,16 @@ JsFunction
         {{ render_downcasted_js_to_rs("js_val", type.err, mut_cx_ref) | indent(8) }}
     }
 }
-{%- elif type.kind == "int" -%}
+{%- elif type.kind == "i32_based" -%}
 {{ js_val }}.value({{ mut_cx_ref }}) as i32
-{%- elif type.kind == "int_based" -%}
-({{ js_val }}.value({{ mut_cx_ref }}) as i32).into()
 {%- elif type.kind == "str" -%}
 {{ js_val }}.value({{ mut_cx_ref }})
 {%- elif type.kind == "str_based" -%}
-match {% if type.custom_from_rs_string -%}
-({{ type.custom_from_rs_string }})({{ js_val }}.value({{ mut_cx_ref }}))
+{% if type.custom_from_rs_string -%}
+let custom_from_rs_string = {{ type.custom_from_rs_string }};
```

Comment:
I'd rather avoid using a let here: having an expression and not a statement here allow for nesting

Reply:
Clippy warns about that